src/dungeon_move_util.c

```diff
@@ -394,8 +394,8 @@ bool8 TryUseChosenMove(struct Entity *attacker, u32 r6, s32 itemId, u32 var_30,
     gUnknown_202F21A = 0;
     gUnknown_202F21C = 0;
     gUnknown_202F220 = 0;
-    if (isLinkedMove && GetEntInfo(attacker)->unk153 <= 3) {
-        GetEntInfo(attacker)->unk153++;
+    if (isLinkedMove && GetEntInfo(attacker)->usedLinkedMovesCounter <= 3) {
```

Comment:
Should this have a define created and be bounded?

Reply:
Mmh, technically this could be < 4, 4 being the max number of moves that can be linked, which is the max number of moves a Pokémon can learn. Does it make sense to be `MAX_MON_MOVES` here?